runtime/hsa-runtime/core/runtime/hsa_api_trace.cpp

```diff
@@ -412,6 +412,7 @@ void HsaApiTable::UpdateAmdExts() {
   amd_ext_api.hsa_amd_profiling_convert_tick_to_system_domain_fn = AMD::hsa_amd_profiling_convert_tick_to_system_domain;
   amd_ext_api.hsa_amd_signal_async_handler_fn = AMD::hsa_amd_signal_async_handler;
   amd_ext_api.hsa_amd_async_function_fn = AMD::hsa_amd_async_function;
+  amd_ext_api.hsa_amd_signal_wait_all_fn = AMD::hsa_amd_signal_wait_all;
```

Comment:
This needs to go at the end (after amd_ext_api.hsa_amd_enable_logging_fn), otherwise it will break the rocprofiler ABI.

Reply:
Done